releases/release-5.1.5.md

```diff
@@ -0,0 +1,161 @@
+    - None [#1152](https://github.com/pingcap/tidb-binlog/issues/1152) @[ti-chi-bot](https://github.com/ti-chi-bot)
+    (dup: release-5.3.2.md > Bug Fixes> TiDB)- Fix the panic issue caused by the `fatal error: concurrent map read and map write` error [#35340](https://github.com/pingcap/tidb/issues/35340)
+    - None [#30402](https://github.com/pingcap/tidb/issues/30402) @[ti-srebot](https://github.com/ti-srebot)
+    - None [#30587](https://github.com/pingcap/tidb/issues/30587) @[ti-srebot](https://github.com/ti-srebot)
+    - None [#37414](https://github.com/pingcap/tidb/issues/37414) @[ti-srebot](https://github.com/ti-srebot)
+    (dup: release-5.4.1.md > Bug Fixes> TiDB)- Fix the issue that TiDB gets the wrong result when using TiFlash to scan tables with empty range although TiFlash does not support reading tables with empty range yet [#33083](https://github.com/pingcap/tidb/issues/33083)
+    (dup: release-5.3.4.md > Bug fixes> TiDB)- Fix the issue that the `avg()` function returns `ERROR 1105 (HY000): other error for mpp stream: Could not convert to the target type - -value is out of range.` when queried from TiFlash [#29952](https://github.com/pingcap/tidb/issues/29952)
+    (dup: release-5.3.4.md > Bug fixes> TiDB)- Fix the issue that `ERROR 1105 (HY000): close of nil channel` is returned when using `HashJoinExec` [#30289](https://github.com/pingcap/tidb/issues/30289)
+    - None [#30923](https://github.com/pingcap/tidb/issues/30923) @[ti-chi-bot](https://github.com/ti-chi-bot)
+    - None [#37932](https://github.com/pingcap/tidb/issues/37932) @[ti-chi-bot](https://github.com/ti-chi-bot)
+    - None [#30923](https://github.com/pingcap/tidb/issues/30923) @[ti-srebot](https://github.com/ti-srebot)
+    (dup: release-5.3.4.md > Bug fixes> TiDB)- Fix the issue that `ERROR 1105 (HY000): close of nil channel` is returned when using `HashJoinExec` [#30289](https://github.com/pingcap/tidb/issues/30289)
+    (dup: release-5.3.4.md > Bug fixes> TiDB)- Fix the issue that TiKV and TiFlash return different results when querying logical operations [#37258](https://github.com/pingcap/tidb/issues/37258)
+    (dup: release-5.3.4.md > Bug fixes> TiDB)- Fix the issue that the `EXECUTE` statement might throw an unexpected error in specific scenarios [#37187](https://github.com/pingcap/tidb/issues/37187)
+    - None [#27125](https://github.com/pingcap/tidb/issues/27125) @[ti-srebot](https://github.com/ti-srebot)
+    (dup: release-5.4.2.md > Bug Fixes> TiDB)- Fix the planner wrong behaviors that occur when `tidb_opt_agg_push_down` and `tidb_enforce_mpp` are enabled [#34465](https://github.com/pingcap/tidb/issues/34465)
+    (dup: release-6.1.1.md > Bug fixes> TiDB)- Fix a bug that TiDB might send coprocessor requests when executing the `SHOW COLUMNS` statement [#36496](https://github.com/pingcap/tidb/issues/36496) @[tangenta](https://github.com/tangenta)
```

Comment:
This is also in the `Improvements` section.

Reply:
Removed tha one and keep this one.